skimage/feature/peak.py

```diff
@@ -173,3 +174,106 @@ def peak_local_max(image, min_distance=1, threshold_abs=None,
         nd_indices = tuple(coordinates.T)
         out[nd_indices] = True
         return out
+
+
+def prominent_peaks(image, min_xdistance=1, min_ydistance=1,
```

Comment:
I'm not familiar with that. Do we have a doc? Didn't see anything in the user guide...